scripts/import.rb

```diff
@@ -13,27 +14,23 @@ def envelope(features)
   }
 end
 
-def description(h)
-  <<~HTML
-    <div class="popup-content">
-      <h2>#{h["title"]}</h2>
-      <p>#{h["address"]}</p>
-      <p>#{h["description"]}</p>
-      <img src="#{h["pic"]}" alt="#{h["title"]}" />
-    </div>
-  HTML
-end
-
 table = CSV.read(ARGV[0], headers: true, col_sep: ";").to_a
 headers = table.shift
 
 table
   .map { |row| Hash[headers.zip(row)] }
-  .map do |h|
+  .map
+  .with_index do |h, n|
+    description = Loofah.fragment(h["description"]).scrub!(:strip).text
```

Comment:
The description field contains html escaped characters and html tags. this removes them.